### DISCUSSION TITLE
Pivot tool for just a piece of the drawing?

### DISCUSSION THREAD

**Discussion by PlumpyPotato** [keiwando/toonsquid#297] [Q&A] [answered]
I wish I could move specific pieces of a drawing/layer using a **pivot**, but I can only move the entire layer instead. 
Separating the parts I want to move into different layers could work, but it's not worth it in my case, especially when I'm not making a puppet-style animation. Is there a way to do it like in the video below? (I used Clip Studio Paint's transform tool to show. CSP's pivot is very small, so it may be a little hard to see it in the video m(_ _)m )

https://user-images.githubusercontent.com/131207750/236724953-924fdf22-60f7-49d8-b0b8-f0ec7f15e57d.mp4

**Comment by PlumpyPotato** [comment]
Maybe I didn't express myself well? I just would like to know if I can have an anchor point/pivot when using the transformation tool to move a part of a drawing, instead of the whole layer/drawing(which is the only case that the pivot shows itself). Toonsquid is relatively new, so I wonder if this feature already exists and I'm being dumb by not finding it yet, or this feature doesn't exist in the app at all.

**Comment by Cat-Butler** [comment]
No, I believe at the moment custom pivot points are only available for an entire Drawing layer, not a selection.
So your current workarounds are either rotate plus translate (move) the selection or copy and paste the selection into a different layer and set the pivot point for that pasted layer (then later merge again if desired).

**Reply by PlumpyPotato** [reply]
So I was not crazy that I couldn't find it (；_・). Thank you for the reply!
*This workaround is going to take forever for frame-by-frame animation, but let's go I guess, it seems there's no other options 
(i v i )*

**Comment by DWrightAnimation** [comment]
I totally get what you're saying. OpenToonz has this in their selection tool as well. If it were to be available in ToonSquid, it would have doing hand drawn animation a bit easier for those who want to save time.

**Reply by PlumpyPotato** [reply]
Toonsquid has so many cool features, that it caught me off guard when I saw this feature wasn't there. It's indeed time-saving, and does wonders when making subtle or looping movements. Crossing my fingers that this gets implemented in the future.
Thank you for the reply!

**Reply by DWrightAnimation** [reply]
I don't know how we missed this, or if it's a new feature in ToonSquid, but if you go into settings, then go to the tools tab, and select "Temporary Pivots Enabled", it'll create pivots for your drawing, any selection of your drawing, pretty much everything we want.

**Comment by DWrightAnimation** [comment, marked as answer]
https://github.com/keiwando/toonsquid/assets/129681998/7ded806b-142a-4417-83f7-4d0427608567 

Here's a video recreating your demonstration in ToonSquid.

**Reply by PlumpyPotato** [reply]
Hmmm it may be a new feature, because I looked through the app a lot at the time, and I don't remember this being available.
Thank you for letting me know, this sure will improve my workflow! :D
